common/static/js/lib/pretty-print.js

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
IMO you should preserve the copyright from the original. Is there really not an NPM-installable library that we can use instead?